packages/core/src/themes/buildFont.ts

```diff
@@ -58,7 +58,7 @@ export default function buildFont(fontFamily: string): Theme['font'] {
     textMicro: {
       fontFamily,
       fontSize: 11,
-      lineHeight: 15 / 11,
```

Comment:
before https://github.com/airbnb/lunar/pull/140/files , this value was `16.5px`. bumping back up to `16px`